app/Models/TouchedApplicationStep.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Models;
+
+use App\Models\BaseModel;
+
+/**
+ * Class Step
+ *
+ * @property int $job_application_id
+ * @property int $step_id
+ * @property boolean $touched
+ * @property \Jenssegers\Date\Date $created_at
+ * @property \Jenssegers\Date\Date $updated_at
+ *
+ * @property \App\Models\JobApplication $job_application
+ * @property \App\Models\Lookup\JobApplicationStep $job_application_step
+ */
+class TouchedApplicationStep extends BaseModel
+{
+    protected $casts = [
+      'job_application_id' => 'int',
+      'step_id' => 'int',
+      'touched' => 'boolean',
+    ];
+
+    protected $fillable = [
+      'job_application_id',
+      'step_id',
```

Comment:
I don't these two should be fillable. They should be immutable after the row is created.